README.md

```diff
@@ -132,16 +132,17 @@ Load
 
 **Note about KAFKA benchmark:**
 
-Command line example:
+Command line example of KAFKA benchmark:
 ```
 ./bin/kafka-run-class.sh \
 org.apache.kafka.tools.ProducerPerformance --throughput=-1 \
 --topic=test-one \
 --num-records=50000000 \
---record-size=100 \
+--record-size=1 \
 --producer-props bootstrap.servers=localhost:9092 \
-buffer.memory=67108864 \
+buffer.memory=33554432 \
 batch.size=8196
+
```

Comment:
It's expected that the results would change if the input configuration is altered. Did they actually?